### ISSUE TITLE
[EP]V6 Threaded Heatsink Variant

### ISSUE THREAD

**Issue by LucienWells** [EVA-3D/eva-main#53]
## Problem description
The addition of a variant of the E3D V6 hotend mount that allows for use of the V6 threaded heatsink.

## Describe the solution you'd like

Addition of an alternative EVA2.x hotend mount to allow for the use of the V6 threaded heatsink, rather than the standard V6 groove-mount.

### Proposed solution

- E3D sells a [threaded heatsink](https://e3d-online.com/collections/upgrades-v6/products/v6-threaded-heatsink) upgrade for the V6 hotend, which does away with the groove-mount.
- I believe [this is the associated E3D technical drawing](https://e3d-online.dozuki.com/Document/PqLiSCu1HbKSCBxy/V6-175-SINK-SCREW.pdf).
- Popular makers of V6 clones have also made [threaded V6 variants available](https://www.aliexpress.com/item/32889029983.html), including with a locking nut.

### Considered Alternatives

- One option would be to make the locking mechanism part of the EVA mount, such that the threaded V6 screws into the EVA mount itself.
- An alternative would be to incorporate a seperate locking nut per the original Voron Mini Afterburner released with the Voron v0 (and still available with the v0.1 for Bowden style setups). 
- Relevant CAD files, including for the threaded V6, are [available from the Voron GitHub](https://github.com/VoronDesign/Voron-0/tree/Voron0.0/VORON-0/CAD).

### Additional context

I appreciate this is probably a low priority given the preponderance of groove-mount vs threaded mount V6s, but the threaded mount is superior in many ways and it would be nice to see this incorporated within the EVA ecosystem.

**Comment by pkucmus** [comment]
This is one of the best made ticket here so far, thank you!
Sadly, you are right about the priority aspect the reason for that is that I'm currently working on V-minion. Another thing is that it's the most hard to achieve / design mount there is so it will require a lot of work and testing. Additionally there's the fact that V6 is the worse hotend of the supported bunch I'm not sure we want to invest much into it.

TL;DR
I'm not saying no, but it will not happen soon and when the time comes we'll see if there's still the need for this.

**Comment by 2904nando** [comment]
For the Github rats (like me searching for this at 3AM) searching for a solution to use the threaded variant of the e3d V6 to use it with a bowden tube, I did that by modifying the Universal Front directly from the Eva homepage and adding a negative volume (directly in PrusaSlicer) to get a bigger hole xD

Is it perfect? No
Is it extremely centered? No
Did I even remove the unused direct drive mounting holes? No
Did I even try to modify the step files? No
**Did it work? Hell yeah it did**

<img width="1174" height="821" alt="Image" src="https://github.com/user-attachments/assets/f9b5df4a-0944-42f8-9251-4e3f78255044" />
